### ISSUE TITLE
Plot asset EAD over time, scenarios in sidebar

### ISSUE THREAD

**Issue by tomalrussell** [nismod/infra-risk-vis#17]
Plot line chart in asset sidebar, EAD against epoch, different colours for RCPs, separate plot per hazard.

Example vega-lite spec - to be constructed from asset attributes:
```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
  "width": 300,
  "height": 200,
  "data": {"values": [
    {"rcp":     "2.6", "epoch": "2050", "ead":  5210000},
    {"rcp":     "2.6", "epoch": "2100", "ead":  7520000},
    {"rcp":     "baseline", "epoch":    "2010", "ead":  4250000},
    {"rcp":     "4.5", "epoch": "2030", "ead":  4800000},
    {"rcp":     "4.5", "epoch": "2050", "ead":  5250000},
    {"rcp":     "4.5", "epoch": "2100", "ead":  8100000},
    {"rcp":     "8.5", "epoch": "2030", "ead":  4850000},
    {"rcp":     "8.5", "epoch": "2050", "ead":  5610000},
    {"rcp":     "8.5", "epoch": "2100", "ead":  10500000}
  ]},
  "mark": {
    "type": "line",
    "point": {
      "filled": true
    }
  },
  "encoding": {
    "x": {"field": "epoch", "timeUnit": "year", "title": "Year"},
    "y": {"field": "ead", "type": "quantitative", "title": "Expected Annual Damages"},
    "color": {"field": "rcp", "type": "nominal", "title": "RCP"}
  }
}
```

**Comment by mz8i** [comment]
Some questions about requirements:
- do we want to replace the current risk section table with this chart or show both at the same time? The contents of the two are roughly the same
- what mechanism do we want for hazard type selection? Something like a dropdown? Or do we want to always produce charts for all types present, and put them one under the other?

Some things to consider tweaking (the current WIP is at https://github.com/nismod/infra-risk-vis/pull/25):
- colour scales: using the automatic categorical colours is OK, but we could try to move away from colours to avoid confusion with the map colour scales, and for example use grays of different darkness, e.g. the higher the RCP, the darker the gray
- layout, size - especially could consider tweaking height
- is a line chart the best choice here? One problem is that the different RCPs don't cover all the epochs, so the charts are quite fragmented, or sometimes display just one point anyway. Maybe this will visually improve once we have more data points calculated and included in the features

**Comment by tomalrussell** [comment]
Closed by #17
- I like having both the table and chart, for both the visual overview and the ease of reading numbers  - though could switch so the chart comes first above the table.
- I think the dropdown works well.
- colour scales and layout work fine for me
- line chart should look better for fluvial/surface with more results
